service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsController.java

```diff
@@ -72,6 +77,10 @@ public AboutToStartOrSubmitCallbackResponse handleMidEvent(@RequestBody Callback
             }
         } catch (FeeRegisterException ignore) {
             data.put("displayAmountToPay", NO.getValue());
+            applicationEventPublisher.publishEvent(new FailedPBAPaymentEvent(callbackRequest,
+                requestData.authorisation(),
+                requestData.userId(),
+                C2_APPLICATION));
```

Comment:
It's the same case as in case submission.

Reply:
@pd-dudkiewicz but then we would never send notification in submitted event if displayAmountToPay was false? because ..

if (featureToggleService.isPaymentsEnabled() && displayAmountToPay(caseDetails)) {

// try call payments api
// catch send notification if failed

}

Reply:
Chatted about alternatives removing this.